prescriptions/ji_/jinja2/setuptools46.yaml

```
@@ -0,0 +1,23 @@
+units:
+  steps:
+  - name: Setuptools46xJinja2Step
+    type: step.Group
+    should_include:
+      adviser_pipeline: true
+    match:
+    - group:
+      - package_version:
+          name: setuptools
+          version: ">46,==3.0,==3.0.1,==3.0.2"
+          index_url: https://pypi.org/simple
+      - package_version:
+          name: jinja2
+          version: ">=2.0,<=2.6"
+          index_url: https://pypi.org/simple
+    run:
+      stack_info:
+      - &stack_info
```

This anchor is not used:

```suggestion
      - 
```

---

Thanks! :+1: